setup.py

```diff
@@ -1,18 +1,17 @@
 import setuptools
 import setuptools.command.install
-import setuptools.command.develop
 from pathlib import Path
 
-class PostInstallCommand(setuptools.command.install.install,
-                         setuptools.command.develop.develop):
```

Comment:
The issue was here: this command cannot be both the develop command and the install comment.

Reply:
This code actually also triggered the develop command when running the install command.